.claude/skills/preflight/SKILL.md

```
@@ -143,6 +143,18 @@ ${CLAUDE_SKILL_DIR}/scripts/fetch-review-threads.sh "$owner" "$repo" "$pr_number
 ```
 Report what's there — CodeRabbit threads with severity, human threads, review decision.
 
+### Prior review deduplication
+
+When the PR has prior preflight review threads, classify them to avoid re-posting dismissed suggestions on subsequent runs. Extract the PR author from the metadata gathered in Step 1 and run:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)